backend/tests/api/test_api_engagement.py

```diff
@@ -0,0 +1,140 @@
+#
```

Comment:
Tester vi ikke nu igen en stor dict-litteral og ender derved med at grave hullet endnu dybere hvis vi får brug for at ændre i test-date. Er det ikke mulit at teste de enkelte felter et ad gangen. Her tester vi eksempelvis den specifikke uuid for job-function 4 gange.

Reply:
Du har helt ret. Jeg reducerer testen til reelt set kun at teste at forspørgsler til de enkelte endpoints gik godt, uden at binde mig for meget op på den specifikke struktur i testdata. - Den slags tester vi andre steder!